### PR TITLE
Move highspeed kubemark args to common and reduce compaction for all kubemarks

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env [hunk 1]
```diff
@@ -4,3 +4,6 @@ ENABLE_BIG_CLUSTER_SUBNETS=true
 # Increase throughput in master components.
 SCHEDULER_TEST_ARGS=--kube-api-qps=100 --kube-api-burst=100
 CONTROLLER_MANAGER_TEST_ARGS=--kube-api-qps=100 --kube-api-burst=100
+
+# Increase throughput for Kubemark master components.
```
Comment (shyamjvs) [inline jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env]: I don't know if that's going to be a big deal - having both kubemark and gce highspeed in same file means that the base cluster in kubemark will also use the increased scheduler and c-m qps. They may potentially be affected.

Comment (wojtek-t) [reply jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env]: Yes - I was aware of that. I don't think it matters.

Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env]: Ok, let's see.

**Pull request merged**